DESCRIPTION

```diff
@@ -1,8 +1,8 @@
 Package: BASiCS
 Type: Package
 Title: Bayesian Analysis of Single-Cell Sequencing data
-Version: 2.1.16
-Date: 2020-10-22
+Version: 2.2.1
```

Comment:
Should we 2.3.1 to match version numbers in devel. We can use 2.2.1 when creating the fix for the release branch.

Reply:
This is the fix for devel - I pulled from bioc

Reply:
Maybe that was from an old version? Current version in devel is 2.3.0: http://bioconductor.org/packages/devel/bioc/html/BASiCS.html

Reply:
Sorry typo. This is meant to be the release patch - hence the target branch

Reply:
The `RELEASE_3_12` branch is 2.2.0

Reply:
ok, got it! Sorry, I didn't read the PR properly. I will merge now.